packages/client-admin/src/pages/projectsettings.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Card, CardContent, CardHeader, FormLabel, IconButton } from '@mui/material';
+import { Formik, Form, Field } from 'formik';
+import { TextField, Icon as MuiIcon } from '@mui/material';
+import { useState } from 'react';
+import { Icon } from '@components/icon';
+import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
+import { useGetProjectQuery } from '@graphql/project/project';
+import { useUpdateProjectMutation } from '@graphql/project/project';
+import { useAuth } from '../context/auth.context';
+
+const IconPreview = (props: any) => {
+  const { icon, size, imageSize } = props;
+
+  return (
+    <MuiIcon fontSize={size}>
+      <img src={icon} alt="icon" style={{ width: imageSize, height: imageSize }} />
+    </MuiIcon>
+  );
+};
+
+export const ProjectSettings = () => {
+  const { token, decoded_token, setToken } = useAuth();
```

Comment:
Just need decoded Token
```suggestion
  const { decoded_token } = useAuth();
```